python/lsst/analysis/ap/zooniverseCutouts.py

```diff
@@ -521,18 +699,32 @@ def run_cutouts(args):
     )
 
     butler = lsst.daf.butler.Butler(args.repo, collections=args.collections)
-    data = select_sources(butler, args.instrument, args.n,
-                          sqlitefile=args.sqlitefile,
-                          postgres_url=args.postgres_url,
-                          namespace=args.namespace)
+    apdb_query = _make_apdbQuery(butler,
+                                 args.instrument,
+                                 sqlitefile=args.sqlitefile,
+                                 postgres_url=args.postgres_url,
+                                 namespace=args.namespace)
+    data = select_sources(apdb_query, args.limit)
 
     config = ZooniverseCutoutsConfig()
     if args.configFile is not None:
         config.load(os.path.expanduser(args.configFile))
-    config.validate()
     config.freeze()
-    cutouts = ZooniverseCutoutsTask(config=config)
-    cutouts.run(data, butler, args.outputPath)
+    cutouts = ZooniverseCutoutsTask(config=config, output_path=args.outputPath)
+
+    getter = select_sources(apdb_query, args.limit)
+    if not args.all:
```

Comment:
Might be worth a code comment here on what testing the truthiness of `args.all` is actually doing here, as it wasn't obvious to me, so might not be for future readers either.